### PR TITLE
Feature/category support for documents labeling#191

### PR DESCRIPTION
**Pull request by iurii-topychkanov**

### CODE REVIEW

File: web/src/components/task/task-sidebar/task-sidebar.tsx [hunk 1]
```diff
@@ -238,43 +240,24 @@ const TaskSidebar: FC<TaskSidebarProps> = ({ onRedirectAfterFinish, jobSettings,
     }`;
     const validationStatus: ValidationPageStatus = isValid ? 'Valid Page' : 'Invalid Page';
 
-    // needed for taskSidebarLabel:
-    useEffect(() => {
-        refetchTaxons();
-    }, [latestLabelsId]);
+    const jobId = useMemo(() => getJobId(), [getJobId]);
```
Comment (sbilevich) [inline web/src/components/task/task-sidebar/task-sidebar.tsx:243]: we already getting job from TaskAnnotator context, maybe we can use it here?

Comment (iurii-topychkanov) [reply web/src/components/task/task-sidebar/task-sidebar.tsx:243]: yeah, it's it, look at line 100. It's just for not calling method getJobId() several times

**Review by sbilevich — APPROVED** [review]

**Comment by iurii-topychkanov** [comment]
Need to fix categories fetching with undefined jobId

File: web/src/api/hooks/categories.ts [hunk 2]
```diff
@@ -227,3 +210,38 @@ export async function fetchInfiniteCategories(
...
+    jobId,
+    filters
+}: DocumentCategoriesByJobProps): DocumentCategoriesByJobResponse => {
+    const categoriesFilter: Filter<keyof Category> = {
+        field: 'type',
+        operator: Operators.EQ,
+        value: 'document'
+    };
+
+    const { data, isError, isLoading } = useCategoriesByJob({
```
Comment (iurii-topychkanov) [inline web/src/api/hooks/categories.ts:237]: idle request if jobId isn't provided

**Pull request merged**